knowledge-base/tooltip-callout-position.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Hide the Tooltip Callout or Change its Position
+description: Learn how to customize the appearance and behavior of the Tooltip small arrow, including adjusting the callout position and hiding it when necessary.
+type: how-to
+page_title: Hide the Tooltip Callout or Change its Position
+slug: tooltip-kb-callout-position
+tags: tooltip, styling, callout
+ticketid: 1628953, 1565205
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Tooltip for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+This Knowledge Base article answers the following questions:
+
+* How to customize the Tooltip callout ?
```

Comment:
You don't need a space between the last word and the ending punctuation in a sentence. Remove for all bullet points. 
```suggestion
* How to customize the Tooltip callout?
```